org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementPower.java

```diff
@@ -71,12 +71,8 @@ public boolean initializeDevice() {
             return false;
         }
 
-        divisor = serverClusterMeasurement.getAcPowerDivisor(Long.MAX_VALUE);
-        multiplier = serverClusterMeasurement.getAcPowerMultiplier(Long.MAX_VALUE);
-        if (divisor == null || multiplier == null) {
-            divisor = 1;
-            multiplier = 1;
-        }
+        determineDivisorAndMultiplier(serverClusterMeasurement);
```

Comment:
Why is this called here - this is at the end of the method, so surely nothing from this method is used after this?

Reply:
Will remove it.